.changeset/perfect-hotels-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sl-design-system/angular': major
+---
+
+Upgrade to Angular 18 and drop support for 17
```

Comment:
is this checked with the cfa's?

Reply:
No, but it's best practice for Angular libraries to support n + n-1. So now that Angular 19 is out, 18 + 19, but not 17.